### ISSUE TITLE
OpenAL SOFT.DeviceClock not exposed -- how to fetch device clock data?

### ISSUE THREAD

**Issue by isolin** [opentk/opentk#1738]
### Description

Unable to use ALC.DeviceClock, its interface seem to be hidden to standard `Get` methods.

### Repro steps

OpenTK Docs list the [DeviceClock](https://opentk.net/api/OpenTK.Audio.OpenAL.ALC.DeviceClock.html) as part of OpenTK. It contains too little information about the exposed interface, but I did not find any relevant constant for neither `ALC.GetInteger` nor `AL.GetSource` to fetch the clock info. `SOFT.DeviceClock` does contain relevant enums, but I can't find a mapping to `AlcGetInteger` or `AlGetSourcei`.

### Expected behavior

The underlying [extension](https://www.openal-soft.org/openal-extensions/SOFT_device_clock.txt) should provide access to the device clock and latency info and all necessary constants and enums should be available and documented.

I found a reference to the extension in [ALTest.cs](https://github.com/opentk/opentk/blob/bf33d2662bc355dd9c4eec7b4a16f5133b49fd0b/tests/OpenALTest/ALTest.cs#L118C17-L118C59):

```cs
if (ALC.DeviceClock.IsExtensionPresent(device))
//...
```
but I am getting `ALC doe not contain a definition for 'DeviceClock'`.

### Related information 

* Ubuntu 22.04
* OpenTK 5.0.0-pre.8
* .NET 8.0

**Comment by NogginBops** [comment]
You are using `5.0.0-pre.8` which is not the latest version. The documentation is updated to be relevant for the latest OpenTK 4.x release.
The extension moved it's namespace in this commit https://github.com/opentk/opentk/commit/1266956f9be56364c52165c8172b7dbca8964e7c which is part of `4.8.0`+ and `5.0.0-pre.9`+.

**Comment by isolin** [comment]
Ah, I used the [deprecated](https://www.nuget.org/packages/OpenTK.OpenAL/) package with `pre.8` being the latest in NuGet. The deprecation warning only shows on the general page, but not for the `pre.8` specific page. So I didn't notice it when installing.

Now I got the [replacement](https://www.nuget.org/packages/OpenTK.Audio.OpenAL/) and it works fine.

Thank you to the OpenTK authors, it is the only solution that work for me when creating kind-of a Linux audio sequencer!
